assets/js/googlesitekit/datastore/user/index.js

```diff
@@ -23,6 +23,7 @@ import Data from 'googlesitekit-data';
 import authentication from './authentication';
 import dateRange from './date-range';
 import disconnect from './disconnect';
+import inputSettings from './input-settings';
```

Comment:
Nitpicking, but let's call this file `user-input-settings.js` since "User Input" is kind of a semi-internal codename for this feature. "input" on its own therefore isn't as clear here.